doc/classes/ViewportTexture.xml

```diff
@@ -7,6 +7,7 @@
 		Provides the content of a [Viewport] as a dynamic [Texture2D]. This can be used to mix controls, 2D game objects, and 3D game objects in the same scene.
 		To create a [ViewportTexture] in code, use the [method Viewport.get_texture] method on the target viewport.
 		[b]Note:[/b] A [ViewportTexture] is always local to its scene (see [member Resource.resource_local_to_scene]). If the scene root is not ready, it may return incorrect data (see [signal Node.ready]).
+		[b]Note:[/b] Some nodes such as [Decal], [Light3D] projector texture and [PointLight2D] texture don't support using [ViewportTexture] directly. To use texture data from a [ViewportTexture] in these nodes, you need to create an [ImageTexture] by calling [method Texture2D.get_image] on the [ViewportTexture]. This conversion is a slow operation, so it should not be performed every frame.
```

Comment:
I don't see why mentioning the properties like these for a few reason:
- "Some nodes such as", but then the sentence refers to the properties specifically;
- All of these nodes only have one property that accepts **Texture2D** (and if there were more they likely wouldn't support **ViewportTexture** either);
- They cannot be translated consistently like this, compared to e.g. `[member Light3D.projector_texture]`.
```suggestion
		[b]Note:[/b] Some nodes such as [Decal], [Light3D] and [PointLight2D] do not support using [ViewportTexture] directly. To use texture data from a [ViewportTexture] in these nodes, you need to create an [ImageTexture] by calling [method Texture2D.get_image] on the [ViewportTexture]. This conversion is a slow operation, so it should not be performed every frame.
```